README.md

```diff
@@ -34,12 +34,10 @@ subtree are not rendered immediately. Instead, when the lock is committed, the
 user-agent processed rendering updates co-operatively, yielding periodically to
 allow script or user interactions to happen.
 
-The visual content of an element's subtree which is locked for display does not
-change. Specifically, if the element already existed in the DOM, then the
-content that was present at the time the lock was acquired remains visibile
-until the lock is committed and the associated promise is resolved. Note that in
-this case, the element itself still responds and updates its own visual state
-(e.g. border); it is only the element's subtree that is locked for display.
+The current consideration is only for an element to be locked before it is
+inserted into the DOM. Display locking can later be extended to lock elements
+already in the DOM, either freezing their visual state, or making them
+invisible.
```

Comment:
We will need the "make them invisible" part soon for unification with Searchable Invisible DOM. Just leave
this paragraph as-is?

Reply:
Done. I removed a small detail, since it doesn't seem relevant.